Makefile

```diff
@@ -86,10 +86,12 @@ ifneq ($(DEBUG_LEVEL), 2)
 OPT += -O2 -fno-omit-frame-pointer
 ifeq (,$(findstring ppc64,$(MACHINE))) # ppc64[le] doesn't support -momit-leaf-frame-pointer
 ifneq ($(MACHINE),armv7l)
+ifneq ($(MACHINE),s390x)
```

Comment:
Thanks. I just rebased. The build_tools/build_detect_platform file still
needs to be patched. I'll submit another pull request.

On Mon, Feb 29, 2016 at 2:07 PM, Igor Canadi <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/facebook/rocksdb/pull/1020#discussion_r54459758>:
>
> > @@ -86,10 +86,12 @@ ifneq ($(DEBUG_LEVEL), 2)
> >  OPT += -O2 -fno-omit-frame-pointer
> >  ifeq (,$(findstring ppc64,$(MACHINE))) # ppc64[le] doesn't support -momit-leaf-frame-pointer
> >  ifneq ($(MACHINE),armv7l)
> > +ifneq ($(MACHINE),s390x)
>
> this should be fixed with #1019
> <https://github.com/facebook/rocksdb/pull/1019>. can you rebase?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/rocksdb/pull/1020/files#r54459758>.
>